src/main/java/Leas_Liudmila/inheritance/AnimalMain.java

```diff
@@ -0,0 +1,34 @@
+package Leas_Liudmila.inheritance;
+
+import java.util.ArrayList;
+
+public class AnimalMain {
+    public static void main(String[] args) {
+        Dog staff = new Dog("Graff", 14, "Gav", "stafford", 12.5f);
+
+        System.out.println(staff.getName() + " " + staff.getAge() + " " + staff.getSound() + " " + staff.getBreed() + " " + staff.getWeight());
```

Comment:
would be cool to override toStrin ginstead of using this construction every time when you need to print object fields